modules/administration-guide/partials/proc_restoring-a-che-instance-by-using-prod-cli-and-a-checluserbackup-custom-object.adoc

```diff
@@ -0,0 +1,37 @@
+[id="restoring-a-{prod-id-short}-instance-by-using-{prod-cli}-and-a-checluserbackup-custom-object_{context}"]
+= Restoring a {prod-short} instance by using {prod-cli} and a CheClusterBackup custom object
+
+You can restore a {prod-short} instance by entering the name of the `CheClusterBackup` custom object as an argument with the `{prod-cli} server:restore` command.
+
+IMPORTANT: If the backup snapshot was taken in an earlier version of Che, then `chectl server:restore --backup-cr-name=__<name_of_custom_object>__` rolls back Che to that earlier version, which is specified in the `CheClusterBackup` custom object.
```

Comment:
```suggestion
IMPORTANT: If the backup snapshot was taken in an earlier version of Che, running `chectl server:restore --backup-cr-name=__<name_of_custom_object>__` rolls back Che to that earlier version, which is specified in the `CheClusterBackup` custom object.
```
I'm not sure whether "rolls back" is the "formal enough term" but it's definitely common. Is it used in Che docs often? Can "revert" be used here with no harm to the meaning?

Reply:
> Is it used in Che docs often? 

No, it is used only in backup / restore sections, but I'd recommend to keep the word `rollback` as there is corresponding command line flag for the `server:restore` command. This way we eliminate confusion about revert is the same as rollback.

Reply:
Thank you for clarifying this. Since `rollback` is a flag, choosing this word over "revert" makes a lot sense indeed.